tests/ssl-opt.sh

```diff
@@ -178,6 +178,14 @@ case "$MBEDTLS_TEST_OUTCOME_FILE" in
         ;;
 esac
 
+# Read boolean configuration options from config.h for easy and quick
+# testing. Skip non-boolean options (with something other than spaces
+# and a comment after "#define SYMBOL"). The variable contains a
+# space-separated list of symbols.
+CONFIGS_ENABLED=" $(<"$CONFIG_H" \
+                    sed -n 's!^ *#define  *\([A-Za-z][0-9A-Z_a-z]*\) *\(/*\)*!\1!p' |
```

Comment:
In my opinion it is more readable in this way:
```suggestion
                    sed -n 's!^ *#define \+\([A-Za-z][0-9A-Z_a-z]*\) *\(//\+\)\?!\1!p' |
```

Reply:
`\+`and `\?` are a GNU extension. They wouldn't work e.g. on FreeBSD or macOS.